src/plugins/teleop/Teleop.cc

```diff
@@ -103,32 +121,47 @@ Teleop::Teleop(): Plugin(), dataPtr(std::make_unique<TeleopPrivate>())
 Teleop::~Teleop() = default;
 
 /////////////////////////////////////////////////
-void Teleop::LoadConfig(const tinyxml2::XMLElement *)
+void Teleop::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
 {
   if (this->title.empty())
     this->title = "Teleop";
 
+  if (_pluginElem)
+  {
+    auto topicElem = _pluginElem->FirstChildElement("topic");
+    if (nullptr != topicElem && nullptr != topicElem->GetText())
+      this->SetTopic(topicElem->GetText());
```

Comment:
Is it possible to write a test for this?

Reply:
Yup, thanks! [b0a7347](https://github.com/gazebosim/gz-gui/pull/440/commits/b0a7347747aa882d4ed784162b33cbdddfb5f019)